base/src/main/scala/com/github/j5ik2o/akka/persistence/s3/base/metrics/MetricsReporter.scala

```diff
@@ -28,10 +27,6 @@ trait MetricsReporter {
   def afterJournalAsyncReadHighestSequenceNr(@unused context: Context): Unit = {}
   def errorJournalAsyncReadHighestSequenceNr(@unused context: Context, @unused ex: Throwable): Unit = {}
 
-  def beforeJournalAsyncUpdateEvent(context: Context): Context = { context }
```

Comment:
Removed unnecessary method definitions of MetricsReporter that are not called from AsyncWriteJournal processing.